### ISSUE TITLE
hold the viewer for different points

### ISSUE THREAD

**Issue by JoltonADsouza** [heremaps/pptk#21]
I am updating my points after every 0.5 seconds using a for loop and I want the viewer window to be the same and not open a new one again and again after every update. How do I do this?

**Comment by samanbey** [comment]
You can clear the viewer and reload points without creating a new window:
v.clear()
v.load(points)

the problem is that the eye position is also reset by using load() - so if anyone knows how to load points without changing the current eye position, let me know :)

**Comment by soulslicer** [comment]
Did anyone figure this out?

**Comment by ozgurovic** [comment]
v.set(lookat = (1,0,-2)) #Camera look-at position
v.set(phi = 0.7) #Camera azimuthal angle (radians)
v.set(theta = 0.7) #Camera elevation angle (radians)
v.set(r = 100) #Camera distance to look-at point

**Comment by samanbey** [comment]
Yes, you can save current viewpoint before reloading points and then reset the saved viewpoint, but before that the default vp will be set for a flash. 
Finally I gave up pptk and used pclpy instead. This problem does not occur there.

**Comment by soulslicer** [comment]
Wow, never even heard of pclpy, it seems great. Will use that instead
